sections/04_study.md

```diff
@@ -59,18 +59,52 @@ particularly notable in this area?*
 *There is not much content here.  Can [@tag:Chen2015_trans_species] be covered
 elsewhere?*
 
-### Cellular phenotypes
-
-*These are primarily imaging-based phenotypes.  We have not surveyed this area
-very comprehensively.  We could decide to not make imaging a primary focus,
-refer to existing reviews, and mention only a few particularly noteworthy
-representative papers.  Alternatively, we need to expand our literature review
-and summaries immediately if someone wants to be responsible for this
-sub-section.*
-
-*Transfer learning from non-biological datasets to biological imaging
-data could fit here, and that does seem like an important topic.  Or
-transfer learning could be a more general topic for the Discussion section.*
+### Morphological phenotypes
+
+A field poised for dramatic revolution by deep learning is
+bioimage analysis - and not in the way you might first think.
```

Comment:
I see your point! Esp if you're trying to formalize the style, I'd suggest just deleting " - and not in the way you might first think" because indeed it is a little too tough to explain and it doesn't become clear until the next paragraph which is too long for someone to be confused. I'm going to try to figure out how to make this edit myself but if I fail, someone else should!